lib/matplotlib/offsetbox.py

```diff
@@ -1270,8 +1275,24 @@ def __init__(self, offsetbox, xy,
             the offset box w.r.t. the *boxcoords*.
             The lower-left corner is (0, 0) and upper-right corner is (1, 1).
 
+        bboxprops : dict, optional
+            A dictionary of properties to set for the annotation bounding box,
+            for example *boxstyle* and *alpha*.  See `.FancyBboxPatch` for
+            details.
+
+        arrowprops: dict, optional
+            Arrow properties, see `.Annotation` for description.
+
+        fontsize: float or str, optional
+            Translated to points and passed as *mutation_scale* into
+            `.FancyBboxPatch` to scale attributes of the box style (e.g. pad
+            or rounding_size).  Passed here as *fontsize* for the convenience
+            of using the named sizes.  If not given, :rc:`legend.fontsize` is
```

Comment:
```suggestion
            or rounding_size).  If not given, :rc:`legend.fontsize` is
```
I'd leave this out. I think it's not really helpful for understanding (or even technically necessary to provide the convenience).

If you want to comment on the name, it should be more in the direction 

> the name is chosen in analogy to [...*] where fontsize defines the mutation scale as well.

\* To be checked what exactly: Text/Annotation/Legend/FancyBboxPatch.

Reply:
Looks like it's in `Text`
https://github.com/matplotlib/matplotlib/blob/57710dba1ac30be50e2c6bdaa691036a3354ca4a/lib/matplotlib/text.py#L582